cli.ts

```diff
@@ -168,6 +170,33 @@ async function cli(args: ParsedArgs) {
 
   logResponse(response)
 
+  if (isEqual(selectedCommand, ["connect-webviews", "create"])) {
```

Comment:
use the response instead of the command

```suggestion
  if (response.data.connect_webview) {
```